data/wanderer/wanderers start.txt

```diff
@@ -406,26 +349,47 @@ mission "Wanderers Conversation"
 	landing
 	source "Vara K'chrai"
 	to offer
-		has "Visit Wanderers Again: done"
+		or
+			has "event: wanderer translator available"
+			# For users before wanderer translator:
+			has "language: Wanderers"
+			has "event: wanderer technology available"
 	on offer
+		event "wanderer technology available"
 		set "license: Wanderer Outfits"
-		log "Spoke with a Wanderer ambassador named Iktat Rek, using the translation device. Received permission to purchase some of their technology, but not their ships."
+		log "Received a translation device and used it to speak with a Wanderer ambassador named Iktat Rek. They granted permission to purchase some of their technology, but not their ships."
```

Comment:
```suggestion
		log "Received a translation device and used it to speak with a Wanderer ambassador named Iktat Rek. He granted me permission to purchase some Wanderers technology, but not their ships."
```

This wording makes more sense to me, as it is explicit about where the permission came from.